redbeams/build.gradle

```diff
@@ -71,7 +71,8 @@ dependencies {
   compile project(':cloud-common')
   compile project(':auth-connector')
   compile project(':core-api')
-  compile project(':core-model')
+  compile project(':workspace-authorization')
```

Comment:
This module is in the repo just to store the old code but I think we do not need it (just talked to Attila Palfi about it).

Reply:
OK, I'll find out more about this.